src/content/blog/release-announcement-0.2.0/index.mdx

```diff
@@ -0,0 +1,58 @@
+By breaking down the portfolio into waves, we enable a more sophisticated process that can automate the calculation of these waves based on various criteria. 
+This feature empowers users to scale adoption efforts effectively, ensuring a seamless and efficient migration experience.
+
+### :sparkles: Jira Integration:
+Jira Integration is a significant development in response to a key concern raised by top managers involved in modernization and migration initiatives. 
+Understanding the overall project progress is crucial, and recognizing that most organizations utilize their task management tools, 
+we have enabled Konveyor to integrate seamlessly with these existing systems to leverage their capabilities.
+Our approach entails creating tasks or issues within the respective task/issue managers to represent the work associated with each application in the portfolio. 
+Konveyor will then retrieve real-time information about the current status of each application, 
+providing comprehensive project visibility. We have designed this integration to be flexible and adaptable, 
+ensuring that it remains independent of any specific task/issue manager peculiarities on the Konveyor side. 
+
+### Minor improvements:
+These enhancements aim to refine and enhance the overall user experience of our platform. Some of the notable improvements are:
+
+:sparkles: ServiceMonitor for hub metrics: We have introduced ServiceMonitor functionality to capture and monitor hub metrics, 
+providing valuable insights into the performance of your system.
+
+:sparkles: Expose Hub metrics: With this update, we have exposed hub metrics, allowing users to access and utilize metrics for monitoring and analysis purposes.
+
+:sparkles: Make Ingress Creation Optional: This update allows Ingress creation optional, providing flexibility and control over your deployment configurations.
+
+:sparkles: Support for Amazon Load Balancer Ingress controller: Our platform now supports the Amazon Load Balancer Ingress controller, 
+enabling Konveyor to be easily deployed on EKS clusters.
+
+:sparkles: Provide openshift-oauth as an authentication option: As part of our ongoing efforts to enhance authentication capabilities, 
+we now offer openshift-oauth as an additional authentication option, providing more choices to meet your specific requirements.
+
+:sparkles: Make rwx_supported false by default: In this release, we have set the default value of rwx_supported to false.
+
+These minor improvements reflect our commitment to continuously refining and enhancing our platform to deliver an optimized and user-friendly experience. 
+We value your feedback and encourage you to explore these enhancements as you leverage our latest release. 
+You can install Konveyor 0.2.0 from [community operators](https://operatorhub.io/operator/konveyor-operator). 
+If you encounter any issues or have suggestions for features, we encourage you to open an issue in the <>
```

Comment:
@jwmatthews Do you know which repo should I include for the issues/enhancements? Like single point of contact repo that we can use for every release and standardize.

Reply:
For issues: https://github.com/konveyor/tackle2-operator/issues
For enhancements/RFEs:  https://github.com/konveyor/enhancements/issues


I'm suggesting we standardize on the operator repo as a place anyone can file an issue against Konveyor then we will triage and reassign as needed this way users do not need to worry about knowing the layout of multiple repos.

When it comes to RFEs, I think it's fine for a user to file something under the operator repo, then we can triage and move to the enhancements repo.

Reply:
Added operator repo link. Thanks @jwmatthews